docs/docs/api/Strapi.mdx

```diff
@@ -596,3 +603,9 @@ TODO
 TODO
 :::
 
+### `validators`
+
+:::info
+TODO
```

Comment:
Is this intentional or forgotten?

Reply:
Intentional. The `sanitizers` is also a TODO, but I didn't want this to get lost when someone finally gets around to documenting everything in the strapi object.